src/webui/src/components/Modal/Compare.tsx

```diff
@@ -20,7 +20,7 @@ class Compare extends React.Component<CompareProps, {}> {
         super(props);
     }
 
-    intermediate = () => {
+    intermediate = (): any => {
```

Comment:
what is the returned object type?

Reply:
It's hard to ensure its type. I try `HTMLElement` `Element` `HTMLDivElement`. all tooltip this error: `Type 'Element' is missing the following properties from type 'HTMLDivElement': align, addEventListener, removeEventListener, accessKey, and 238 more.ts(2740)`

Reply:
Have you tried `React.ReactNode`?